oval_graph/_xml_parser_oval_scan_definitions.py

```diff
@@ -30,7 +30,9 @@ def _build_graph(self, tree_data):
             graph['node'].append(self._build_node(tree, "Definition"))
         return graph
 
-    def _str_to_bool(self, s):
+    # rework this
```

Comment:
Without further explanation, comments like this will confuse even the person who wrote them.

Reply:
This is my mark for rework in the next PR.